ruby_event_store/spec/mappers/default_spec.rb

```diff
@@ -0,0 +1,46 @@
+SomethingHappened = Class.new(RubyEventStore::Event)
+
+module RubyEventStore
+  module Mappers
+    RSpec.describe Default do
+      let(:data)   {{some_attribute: 5}}
+      let(:metadata)     {{some_meta: 1}}
+      let(:domain_event) { SomethingHappened.new(data: data, metadata: metadata) }
+
+      specify '#event_to_serialized_record returns YAML serialized record' do
+        record = subject.event_to_serialized_record(domain_event)
+        expect(record).to            be_a SerializedRecord
+        expect(record.id).to         eq domain_event.event_id
+        expect(record.metadata).to   eq YAML.dump(metadata)
+        expect(record.data).to       eq YAML.dump(data)
+        expect(record.event_type).to eq SomethingHappened
+      end
+
+      specify '#serialized_record_to_event returns event instance' do
+        record = SerializedRecord.new(
+          id:         domain_event.event_id,
+          data:       YAML.dump(data),
+          metadata:   YAML.dump(metadata),
+          event_type: SomethingHappened.name
+        )
+        event = subject.serialized_record_to_event(record)
+        expect(event).to              be_a SomethingHappened
+        expect(event.event_id).to     eq domain_event.event_id
+        expect(event.data).to         eq(data)
+        expect(event.metadata).to     eq(metadata)
+      end
+
+      specify '#serialized_record_to_event its using events class remapping' do
+        subject = described_class.new(events_class_remapping: {'EventNameBeforeRefactor' => 'SomethingHappened'})
+        record = SerializedRecord.new(
+          id:         domain_event.event_id,
+          data:       YAML.dump(data),
+          metadata:   YAML.dump(metadata),
+          event_type: "EventNameBeforeRefactor"
+        )
+        event = subject.serialized_record_to_event(record)
+        expect(event).to be_a SomethingHappened
+      end
+    end
+  end
+end
```

Comment:
It would be nice to test with another serializer to document that it is possible to use it in: `def initialize(serializer: YAML` 

I guess mutant is missing a mutation similar to:

```ruby
def initializer(serializer: Object.new)
  serializer = YAML
end
```

that would show that you never use anything other than the default so why is it a method parameter :)